src/main/resources/templates/jenkins/swift/regularRuns/Jenkinsfile

```diff
@@ -50,7 +50,7 @@ pipeline {
                     mkdir results
                     if [ -e assignment/tests.xml ]
                     then
-                        sed -i 's/<testsuites>//g ; s/<\/testsuites>//g' assignment/tests.xml
+                        sed -i 's/<testsuites>/<testsuite>/g ; s/<\/testsuites>/<\/testsuite>/g' assignment/tests.xml
```

Comment:
That change makes sense since the Jenkins plugin got the ability to read nested testsuites a few versions ago.
I’m not sure how/if that worked before, though, since the XML reader just [assumed a single test suite](https://github.com/ls1intum/jenkins-server-notification-plugin/blob/7c2995220ad4ac87006011a60338cc0b271e6268/src/main/java/de/tum/in/www1/jenkins/notifications/SendTestResultsNotificationPostBuildTask.java#L120). With the previous `sed` replacement, now the XML contains multiple root elements, i.e. a `List<TestSuite>`. For Python and Haskell (nearly) identical replacements seem to be fine.